Gifski/VideoValidator.swift

```
@@ -184,6 +184,20 @@ struct VideoValidator {
 			return .failure
 		}
 
-		return .success(newAsset, newVideoMetadata)
+		// Trim asset
+		do {
+			let trimmedAsset = try newAsset.trimBlankFrames()
+
+			return .success(trimmedAsset, newVideoMetadata)
+		} catch {
```

You should not ignore the error. It should be included in the alert.

---

Fixed in 50532fc.